src/com/gfk/jenkins/components/DockerBuildImage.groovy

```diff
@@ -0,0 +1,27 @@
+package com.gfk.jenkins.components
+
+class DockerBuildImage implements Serializable {
+    def steps
+    DockerBuildImage(steps) { this.steps = steps }
+    def buildMe(buildPath) {
+        def dockerImage
+        def buildParams = [
+                "--no-cache",
+                "--rm",
+                "--build-arg TIME=${steps.env.TIME}",
+                "--build-arg REPO=${steps.env.REPO}",
+                "--build-arg HUDSON_URL=${steps.env.HUDSON_URL}",
+                "--build-arg GIT_URL=${steps.env.GIT_URL}",
+                "--build-arg SHORT_SHA=${steps.env.SHORT_SHA}"
+        ]
+        def path = buildPath ? "${buildParams.join(" ")} -f ${buildPath} ." : "${buildParams.join(' ')} ."
+
+        dockerImage = steps.docker.build(
+                "${Configuration.AWS_ACCOUNT}.dkr.ecr.${Configuration.AWS_REGION}.amazonaws.com/${steps.env.PROJECT.toLowerCase()}/${steps.env.REPO}:${steps.env.TAG}",
```

Comment:
You can define a new variable with the docker tag string to avoid use same log string several times.